apps/web/app/(app)/environments/[environmentId]/lib/project.ts

```diff
@@ -0,0 +1,51 @@
+import { validateInputs } from "@/lib/utils/validate";
+import { Prisma } from "@prisma/client";
+import { cache as reactCache } from "react";
+import { prisma } from "@formbricks/database";
+import { ZString } from "@formbricks/types/common";
+import { DatabaseError } from "@formbricks/types/errors";
+import { TMembership, ZMembership } from "@formbricks/types/memberships";
+
+export const getUserProjects = reactCache(
```

Comment:
inconsistent naming: you named the one function `getOrganizationsByUserId` and this one `getUserProjects`. To stay consistent you should also name this `getProjectsByUserId`